### PR TITLE
Correct typos and phrasing in the third-party packages sections

### PR DESCRIPTION
**Pull request by kwitaszczyk**
This PR continues the review from https://github.com/CTSRD-CHERI/cheribsd-getting-started/pull/12.

### CODE REVIEW

File: src/packages/limitations.md [hunk 1]
```diff
@@ -2,37 +2,38 @@
...
+In such cases, a corresponding hybrid ABI package might be better suited than
+the CheriABI package.
 
 We expect the number of available and tested packages to grow with future
-CheriBSD releases but it will take quite some time until we will be able to
-provide a CheriABI package repository that would include all packages currently
-provided with the hybrid ABI package repository.
+CheriBSD releases, but it will take quite some time until we are able to provide
+a CheriABI package repository that includes all packages currently provided in
+the hybrid ABI package repository.
```
Comment (brooksdavis) [inline src/packages/limitations.md:27]: "all packages ..." probably isn't a goal. Not sure how best to temper that or if we should bother.

**Review by brooksdavis — APPROVED** [review]

**Review by brooksdavis — APPROVED** [review]
One comment that I don't think we need to fix.

File: src/packages/limitations.md (hunk 1 above)
Comment (kwitaszczyk) [reply src/packages/limitations.md]: Maybe more broadly like this (in the current PR version)?
> (..) provide a CheriABI package repository that covers the same functionalities which are included in the hybrid ABI package repository as of today.

Comment (brooksdavis) [reply src/packages/limitations.md]: works for me

**Comment by kwitaszczyk** [comment]
Let's merge it now, and we can improve it separately, if needed.

**Pull request merged**